### DISCUSSION TITLE
Make command not found for Creating an importer

### DISCUSSION THREAD

**Discussion by mohaaosman** [filamentphp/filament#10034] [Help] [answered]
Hello,

The command `php artisan make:filament-importer Product --generate` shows `Command "make:filament-importer" is not defined.`. following the instructions mentioned in this link `https://filamentphp.com/docs/3.x/actions/prebuilt-actions/import#creating-an-importer`

I event check all the available make commands: 
make:cast  
  ⇂ make:channel  
  ⇂ make:command  
  ⇂ make:component  
  ⇂ make:controller  
  ⇂ make:event  
  ⇂ make:exception  
  ⇂ make:export  
  ⇂ make:factory  
  ⇂ make:filament-apex-charts  
  ⇂ make:filament-api-handler  
  ⇂ make:filament-api-service  
  ⇂ make:filament-api-transformer  
  ⇂ make:filament-page  
  ⇂ make:filament-panel  
  ⇂ make:filament-relation-manager  
  ⇂ make:filament-resource  
  ⇂ make:filament-theme  
  ⇂ make:filament-user  
  ⇂ make:filament-widget  
  ⇂ make:form-field  
  ⇂ make:form-layout  
  ⇂ make:import  
  ⇂ make:infolist-entry  
  ⇂ make:infolist-layout  
  ⇂ make:job  
  ⇂ make:listener  
  ⇂ make:livewire  
  ⇂ make:livewire-form  
  ⇂ make:livewire-table  
  ⇂ make:mail  
  ⇂ make:middleware  
  ⇂ make:migration  
  ⇂ make:model  
  ⇂ make:notification  
  ⇂ make:observer  
  ⇂ make:policy  
  ⇂ make:provider  
  ⇂ make:request  
  ⇂ make:resource  
  ⇂ make:rule  
  ⇂ make:scope  
  ⇂ make:seeder  
  ⇂ make:spotlight  
  ⇂ make:table-column  
  ⇂ make:test  
  ⇂ make:view  

Is there some i'm missing?

The composer:

`{
    "name": "laravel/laravel",
    "type": "project",
    "description": "The skeleton application for the Laravel framework.",
    "keywords": [
        "laravel",
        "framework"
    ],
    "license": "MIT",
    "require": {
        "php": "^8.1",
        "bezhansalleh/filament-exceptions": "^2.0",
        "bezhansalleh/filament-shield": "^3.1",
        "blade-ui-kit/blade-heroicons": "^2.1",
        "cheesegrits/filament-phone-numbers": "^1.0",
        "eightynine/filament-excel-import": "^3.0",
        "filament/actions": "^3.0-stable",
        "filament/filament": "^3.0-stable",
        "guzzlehttp/guzzle": "^7.2",
        "husam-tariq/filament-database-schedule": "^2.0",
        "ibrahimbougaoua/filament-sort-order": "^1.1",
        "irazasyed/larasupport": "*",
        "kenepa/multi-widget": "^2.0",
        "laravel/framework": "^10.10",
        "laravel/sanctum": "^3.3",
        "laravel/telescope": "^4.17",
        "laravel/tinker": "^2.8",
        "leandrocfe/filament-apex-charts": "^3.0",
        "orangehill/iseed": "^3.0",
        "owen-it/laravel-auditing": "^13.5",
        "pxlrbt/filament-activity-log": "^1.0",
        "pxlrbt/filament-environment-indicator": "^2.0",
        "pxlrbt/filament-excel": "^2.1",
        "pxlrbt/filament-spotlight": "^1.0",
        "rappasoft/laravel-authentication-log": "^3.0",
        "rupadana/filament-api-service": "^1.0",
        "shuvroroy/filament-spatie-laravel-backup": "^2.0",
        "shuvroroy/filament-spatie-laravel-health": "^2.0",
        "spatie/laravel-activitylog": "^4.7",
        "spatie/laravel-backup": "^8.4",
        "spatie/laravel-permission": "^6.1",
        "stechstudio/filament-impersonate": "^3.5",
        "tapp/filament-authentication-log": "^3.0",
        "tapp/filament-timezone-field": "^3.0",
        "torann/geoip": "^3.0",
        "webbingbrasil/filament-advancedfilter": "^3.0",
        "ysfkaya/filament-phone-input": "^2.1",
        "z3d0x/filament-logger": "^0.6.1"
    },
    "require-dev": {
        "barryvdh/laravel-debugbar": "^3.9",
        "fakerphp/faker": "^1.9.1",
        "laravel/pint": "^1.0",
        "laravel/sail": "^1.18",
        "mockery/mockery": "^1.4.4",
        "nunomaduro/collision": "^7.0",
        "phpunit/phpunit": "^10.1",
        "spatie/laravel-ignition": "^2.0"
    },
    "autoload": {
        "psr-4": {
            "App\\": "app/",
            "Database\\Factories\\": "database/factories/",
            "Database\\Seeders\\": "database/seeders/"
        }
    },
    "autoload-dev": {
        "psr-4": {
            "Tests\\": "tests/"
        }
    },
    "scripts": {
        "post-autoload-dump": [
            "Illuminate\\Foundation\\ComposerScripts::postAutoloadDump",
            "@php artisan package:discover --ansi",
            "@php artisan filament:upgrade"
        ],
        "post-update-cmd": [
            "@php artisan vendor:publish --tag=laravel-assets --ansi --force"
        ],
        "post-root-package-install": [
            "@php -r \"file_exists('.env') || copy('.env.example', '.env');\""
        ],
        "post-create-project-cmd": [
            "@php artisan key:generate --ansi"
        ]
    },
    "extra": {
        "laravel": {
            "dont-discover": []
        }
    },
    "config": {
        "optimize-autoloader": true,
        "preferred-install": "dist",
        "sort-packages": true,
        "allow-plugins": {
            "pestphp/pest-plugin": true,
            "php-http/discovery": true
        }
    },
    "minimum-stability": "stable",
    "prefer-stable": true
}
`

**Comment by leandrocfe** [comment, marked as answer]
Did you run `composer update`
?

**Reply by icarojobs** [reply]
Nice! It solved my problem! Thank you very much!

**Comment by yogigurjar** [comment]
It's not working in laravel 11
![error command "make:import" is not defined. did you mean one of these?)](https://github.com/filamentphp/filament/assets/71335060/7fccbfc6-a356-4583-86a0-a108c7a13592)

**Reply by leandrocfe** [reply]
`php artisan make:filament-importer`
